src/OfflineSearch.xml

```diff
@@ -33,6 +28,11 @@
             <category>Data source</category>
             <description>The default query that is used to filter the list</description>
         </property>
+        <property key="searchEntity" type="string" required="false">
```

Comment:
Why change the order? It is more logical to select the entity name first and then the attribute.

Reply:
Danny requested for it on slack because it is optional